src/plugins/WebOfTrust/util/BackgroundJobFactoryBase.java

```diff
@@ -0,0 +1,64 @@
+package plugins.WebOfTrust.util;
+
+import java.util.ArrayList;
+import java.util.WeakHashMap;
+
+/**
+ * Base class that provides all boilerplate necessary to implement the {@link BackgroundJobFactory}
+ * interface.
+ *
+ * @author bertm
+ */
+public abstract class BackgroundJobFactoryBase implements BackgroundJobFactory {
+    private static final Object PLACEHOLDER = new Object();
+
+    /** Set of all live (i.e. not garbage collected) background jobs created by this instance. */
+    private final WeakHashMap<BackgroundJob, Object> aliveJobSet =
```

Comment:
To speed up the release, I've fixed this myself in commit 4ebe32fbc370552ee8604ce20c337eeaeeecad3f.

-> Done.